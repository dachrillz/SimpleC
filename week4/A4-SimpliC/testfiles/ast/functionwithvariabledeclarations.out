Program
  List
    Function
      IntType
      IdDecl ID="a"
      List
      Block
        List
          Decl
            IntType
            IdDecl ID="x"
            Opt
          Decl
            IntType
            IdDecl ID="y"
            Opt
