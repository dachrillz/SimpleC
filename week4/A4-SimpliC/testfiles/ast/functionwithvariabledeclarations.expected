Program
  List
    Function
      IdDecl ID="a"
      List
      Block
        List
          Decl
            IdDecl ID="x"
            Opt
          Decl
            IdDecl ID="y"
            Opt
